software-delivery-performance-terraform/sumologic.tf

```diff
@@ -0,0 +1,242 @@
+}
+
+# Create/Delete OpsGenie Source
+resource "sumologic_http_source" "opsgenie" {
+  count        = "${var.install_opsgenie}" ? 1 : 0
+  name         = "OpsGenie"
+  category     = var.opsgenie_sc
+  collector_id = sumologic_collector.sdp_collector.id
+}
+
+# Create/Delete Pagerduty Source
+resource "sumologic_http_source" "pagerduty" {
+  count        = "${var.install_pagerduty}" ? 1 : 0
+  name         = "Pagerduty"
+  category     = var.pagerduty_sc
+  collector_id = sumologic_collector.sdp_collector.id
+}
+
+# Create/Delete Github Source
+resource "sumologic_http_source" "github" {
+  count        = "${var.install_github}" ? 1 : 0
+  name         = "Github"
+  category     = var.github_sc
+  fields       = { "_convertHeadersToFields" = "true" }
+  collector_id = sumologic_collector.sdp_collector.id
+}
+
+# Create/Delete Jenkins Source
+resource "sumologic_http_source" "jenkins" {
+  count        = "${var.install_jenkins}" ? 1 : 0
+  name         = "Jenkins"
+  category     = var.jenkins_sc
+  fields       = { "_convertHeadersToFields" = "true" }
+  collector_id = sumologic_collector.sdp_collector.id
+}
+
+# Sumo Logic - Create/Delete folder for installing the applications
+data "sumologic_personal_folder" "personalFolder" {}
+
+resource "sumologic_folder" "folder" {
+  name        = var.app_installation_folder
+  description = "Atlassian Applications"
+  parent_id   = data.sumologic_personal_folder.personalFolder.id
+  depends_on  = [sumologic_collector.sdp_collector]
+}
+
+# Remove slash from Sumo Logic sumo_api_endpoint
+
+locals {
+  sumo_api_endpoint_fixed = replace(var.sumo_api_endpoint, "api/", "api")
+}
+
+# Install Apps
+# Install Jira Cloud
+resource "null_resource" "install_jira_cloud_app" {
+  count      = "${var.install_jira_cloud}" ? 1 : 0
+  depends_on = [sumologic_http_source.jira_cloud]
+
+  provisioner "local-exec" {
+    command = <<EOT
+        curl -s --request POST '${local.sumo_api_endpoint_fixed}/v1/apps/019757ca-3b08-457c-bd15-7239f1ab66c9/install' \
+            --header 'Accept: application/json' \
+            --header 'Content-Type: application/json' \
+            -u ${var.sumo_access_id}:${var.sumo_access_key} \
+            --data-raw '{ "name": "Jira Cloud", "description": "The Sumo Logic App for Jira Cloud provides insights into project management issues that enable you to more effectively plan, assign, track, report, and manage work across multiple teams.", "destinationFolderId": "${sumologic_folder.folder.id}","dataSourceValues": {"jiralogsrc": "_sourceCategory = ${var.jira_cloud_sc}" }}'
+    EOT
+  }
+}
+
+# Install Jira Server
+resource "null_resource" "install_jira_server_app" {
+  count      = "${var.install_jira_server}" ? 1 : 0
+  depends_on = [sumologic_http_source.jira_server]
+
+  provisioner "local-exec" {
+    command = <<EOT
+        curl -s --request POST '${local.sumo_api_endpoint_fixed}/v1/apps/ec905b32-e514-4c63-baac-0efa3b3a2109/install' \
```

Comment:
Same question with curl

Reply:
No Terraform resource for App installation.